metrics/ci_integrations/test/client/buildkite/models/buildkite_organization_test.dart

```diff
@@ -0,0 +1,96 @@
+import 'package:ci_integration/client/buildkite/models/buildkite_organization.dart';
+import 'package:test/test.dart';
+
+void main() {
+  group("BuildkiteOrganization", () {
+    const id = 'id';
+    const name = 'test';
+    const slug = 'slug';
+
+    const organizationJson = {
+      'id': id,
+      'name': name,
+      'slug': slug,
+    };
+
+    const organization = BuildkiteOrganization(
+      id: id,
+      name: name,
+      slug: slug,
+    );
+
+    test(
+      "creates an instance with the given parameters",
+      () {
+        const organization = BuildkiteOrganization(
+          id: id,
+          name: name,
+          slug: slug,
+        );
+
+        expect(organization.id, equals(id));
+        expect(organization.name, equals(name));
+        expect(organization.slug, equals(slug));
+      },
+    );
+
+    test(
+      ".fromJson() returns null if the given json is null",
+      () {
+        final organization = BuildkiteOrganization.fromJson(null);
+
+        expect(organization, isNull);
+      },
+    );
+
+    test(
+      ".fromJson() creates an instance from the given json",
+      () {
+        final organization = BuildkiteOrganization.fromJson(organizationJson);
+
+        expect(organization, equals(organization));
```

Comment:
```suggestion
        final actualOrganization = BuildkiteOrganization.fromJson(organizationJson);

        expect(actualOrganization, equals(organization));
```
Please, apply to all similar tests.